selenium/src/test/java/com/deque/html/axecore/selenium/Axe43xIntegrationTest.java

```diff
@@ -37,19 +37,19 @@
 import org.junit.BeforeClass;
```

Comment:
most of this was prettier

Reply:
This was likely the precommit hook that was have https://github.com/dequelabs/axe-core-maven-html/blob/develop/scripts/pre-commit-hook.sh#L6